packages/server/lib/project_utils.ts

```diff
@@ -0,0 +1,132 @@
+import Debug from 'debug'
```

Comment:
Again, more concerns that do not really need to be defined on `Project`, so I moved them here.